src/Model/Filter/FilterCollection.php

```diff
@@ -74,7 +74,7 @@ public function add($name, $filter, array $options = [])
     protected function _loadFilter($name, $filter, array $options = [])
     {
         if (empty($options['className'])) {
-            $class = Inflector::classify($filter);
```

Comment:
Why this change?

Reply:
Revert and u see: bad code killing the name resolvement for things that are plural or appear as such

Reply:
That's what tests are for :wink:.